footer.js

```diff
@@ -0,0 +1,63 @@
+const repo = "todaysCuriosity";
+const owner = "mrhut10";
+const requestURL =
+  "https://api.github.com/repos/" + owner + "/" + repo + "/contributors";
+
+const contributors = [];
+
+const request = new XMLHttpRequest();
```

Comment:
This one is super low priority, But I'd prefer you use the `fetch()` API over `XMLHttpRequest()`

Reply:
`async `, `await` and `fetch()` are new to me. I want to make these changes, but I'd want to educate myself on these topics first. Maybe in another PR?

Reply:
Um I'm going to say a solid maybe on this, I'm likely not going to be doing any coding in the next 24hr because I'm busy, but after that I'm going to try and get this PR merged. I don't like PR's sitting around for too long. But I'll change this to fetch before I do the merge. 

If you want to make a separate PR that you aim back at this branch before I get to that that would work. But I wouldn't want you wasting your time in the case that you get halfway through it than I make my own changes and merge this branch.